src/pal/src/loader/module.cpp

```diff
@@ -57,8 +57,10 @@ Module Name:
 #include <sys/types.h>
 #include <sys/mman.h>
 
-#if defined(__LINUX__)
+#if defined(__LINUX__) && defined(__GLIBC__)
 #include <gnu/lib-names.h>
+#else
+#define LIBC_SO "libc.so"
```

Comment:
This was a quick change to get it working, not sure of the correct path forward.